### ISSUE TITLE
error when installing gtk

### ISSUE THREAD

**Issue by royalosyin** [dongli/packman#17]
```
[Notice]: Decompress gtk+-2.24.25.tar.xz.
[Notice]: Install package gtkx with compiler set 0.
==> ./configure --prefix=/home/vagrant/.packman/gtkx/2.24.25/0 --disable-dependency-tracking --disable-silent-rules --disable
/vagrant/centos2/software/packman/packages/gtkx.rb:33:in `install': private method `include' called for X11:Class (NoMethodError)
	from /vagrant/centos2/software/packman/framework/command/install.rb:158:in `block in install_package'
	from /vagrant/centos2/software/packman/framework/utils.rb:108:in `work_in'
	from /vagrant/centos2/software/packman/framework/command/install.rb:153:in `install_package'
	from /vagrant/centos2/software/packman/framework/command/install.rb:45:in `block in install'
	from /vagrant/centos2/software/packman/framework/command/install.rb:36:in `each'
	from /vagrant/centos2/software/packman/framework/command/install.rb:36:in `install'
	from /vagrant/centos2/software/packman/framework/command_line.rb:139:in `run'
	from /vagrant/centos2/software/packman/packman:34:in `<main>'
```

**Comment by dongli** [comment]
Please use markdown syntax to format your post. Thanks!

**Comment by dongli** [comment]
Fixed in commit 1af22f5b.
